models/Thermostat.json

```diff
@@ -0,0 +1,28 @@
+{
+  "@context": "dtmi:dtdl:context;2",
+  "@id": "dtmi:azsamples:iotdd:Thermostat;1",
+  "@type": "Interface",
+  "displayName": "Sample thermostat with remote control",
+  "contents": [
+    {
+      "@type": "Component",
+      "schema": "dtmi:azsamples:iotdd:TemperatureSensor;1",
+      "name": "tempSensor1"
+    },
+    {
+      "@type": "Component",
+      "schema": "dtmi:azure:DeviceManagement:DeviceInformation;1",
+      "name": "deviceInfo"
```

Comment:
>deviceInfo [](start = 15, length = 10)

There was an email thread about using a standard name for the DeviceInformation component. I thought it was "deviceInformation" or is that shortened now to "deviceInfo"?

Reply:
The thread was about which casing to use for component names. I've seen deviceInformation but I'd like to use a different word from the last segment of the id, and different from the display Name to enforce the "components/instance" concept.

IMO makes no sense to standardize instance names.

Reply:
I tend to agree with you. I just wanted to mention it in case we were trying to use standard component names.

---
In reply to: [417694840](https://github.com/rido-min/pnp-ppr-sample-spec/pull/2#discussion_r417694840) [](ancestors = 417694840)